### PR TITLE
Updating with changes required for camera system updates

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
## Overview

Updating with changes required forhttps://github.com/XRTK/XRTK-Core/pull/386

## Changes:

- Requires https://github.com/XRTK/XRTK-Core/pull/386

### CODE REVIEW

**Comment by SimonDarksideJ** [comment]
What other PR, this is for the WMR package

**Comment by StephenHodgson** [comment]
https://github.com/XRTK/XRTK-Core/pull/386

I've went ahead and updated that PR with this branch checkout to be included :)

File: XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Controllers/WindowsMixedRealityController.cs [hunk 1]
```diff
@@ -243,10 +243,10 @@ private void UpdateGripData(InteractionSourceState interactionSourceState, Mixed
                         interactionSourceState.sourcePose.TryGetPosition(out currentGripPosition, InteractionSourceNode.Grip);
                         interactionSourceState.sourcePose.TryGetRotation(out currentGripRotation, InteractionSourceNode.Grip);
 
-                        if (MixedRealityToolkit.Instance.MixedRealityPlayspace != null)
+                        if (MixedRealityToolkit.CameraSystem.CameraRig.PlayspaceTransform != null)
```
Comment (StephenHodgson) [inline XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Controllers/WindowsMixedRealityController.cs]: Was this the only controller we missed in the update?

Comment (SimonDarksideJ) [reply XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Controllers/WindowsMixedRealityController.cs]: That I'm aware of, although I didn't validate the Lumin platform (but figured you had :D )

Comment (StephenHodgson) [reply XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Controllers/WindowsMixedRealityController.cs]: Interesting that we're doing the spatial transformation for this controller but not the others.

Comment (SimonDarksideJ) [reply XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Controllers/WindowsMixedRealityController.cs]: I think it is simply leftover from the original implementation, or something possibly to do with the WMR implementation requiring it maybe?

Comment (StephenHodgson) [reply XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Controllers/WindowsMixedRealityController.cs]: @SimonDarksideJ should these be pointing to the `BodyTransform`?

Comment (SimonDarksideJ) [reply XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Controllers/WindowsMixedRealityController.cs]: I'll test and update. in architecture terms, probably.  Ultimately though, everything is at 0, so it wouldn't matter.

**Comment by StephenHodgson** [comment]
I realized this went to master. We gotta be more aware of the branch we're targeting with our pull requests

**Pull request merged**